src/quo2/components/selectors/reactions_selector/view.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo2.components.selectors.reactions-selector.view
+  (:require [quo2.components.selectors.reaction-resource :as reactions.resource]
+            [quo2.components.selectors.reactions-selector.style :as style]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(defn view
+  [{:keys [start-pressed?]}]
```

Comment:
what is start-pressed?

Reply:
Didn't touch this prop. The component seems to be uncontrolled and `start-pressed` is its initial state.